--- migrate-3.6.8/src/main.c	2014-03-26 10:40:08.000000000 -0700
+++ patch-files/main.c	2015-05-15 15:53:14.172610916 -0700
@@ -865,7 +865,7 @@
     //#warning "Windows version is hardcoded to not have more than 5000 populations"
     long temp[5000];
 #else
-  long temp[data->numpop];
+  long temp[5000];
 #endif
   //use order of populations {1,2,3,4,5,....} , do not start with zero!!
   //reset to options->newpops={1,1,2,1,2,3,4,...}
